docs/src/documentation/03-components/04-overlay/drawer/03-accessibility.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/drawer/accessibility/
+---
+
+## Accessibility
+
+## Drawer
+
+The Drawer component has been designed with accessibility in mind.
+
+To ease keyboard navigation, when opening a drawer, the focus is moved to the first focusable element inside the drawer.
+
+It should not be possible to focus anything outside of the drawer while it is open, ensuring a focused user experience.
+
+When closing the drawer, **the focus should be moved back to the element that triggered the drawer**.
+This is not handled automatically, so make sure to implement this behavior in your application by managing focus properly.
+
+### Keyboard interaction
+
+The Drawer component handles the following keyboard interactions:
+
+- **Escape** key closes the drawer
+- **Tab** key cycles through focusable elements within the drawer
+- **Shift+Tab** navigates backward through focusable elements
+
+### ARIA attributes
+
+The Drawer component accepts ARIA attributes to ensure it's accessible to users of assistive technologies. You can provide these attributes as described below:
+
+| Name      | Type     | Description                                                                                                                 |
+| :-------- | :------- | :-------------------------------------------------------------------------------------------------------------------------- |
+| ariaLabel | `string` | Text that labels the drawer content. Think of it as the title of the drawer. This should be used if `title` is not defined. |
+
+If you provide a `title` prop, it is automatically used as the drawer's `aria-label`.
+However, if you also provide a `ariaLabel` prop, it will take precedence over the `title` prop.
+
+### Close button
+
+The Drawer component includes a close button that can be displayed in the header. It's important to use the `labelHide` prop to provide an accessible label for this button.
+
+The default value is "Hide", but you should consider providing a more descriptive label, especially for internationalization purposes.
+
+### Toggle element
+
+When implementing a toggle element to open and close the drawer, it's essential that the element uses the `aria-expanded` attribute to indicate whether the drawer is open (`true`) or closed (`false`).
+This informs assistive technologies about the current state of the drawer.
```

Comment:
Maybe we can also mention `aria-controls` for that element?

Reply:
I understand what you mean, I think it's the same for tooltip with aria-describedby.

But in Drawer docs we say this:

<img width="845" alt="Screenshot 2025-04-17 at 10 33 19" src="https://github.com/user-attachments/assets/bc052992-2244-4455-adc4-0272cf70a1de" />

So is it or is it not always in the DOM? 🤔

Reply:
I stand corrected, because I worked on this component recently 😅 I will then include mention to aria-controls